x-pack/plugins/maps/public/classes/sources/mvt_single_layer_vector_source/mvt_field_config_editor.test.tsx

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+jest.mock('../../../kibana_services', () => ({}));
+
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import { MVTFieldConfigEditor } from './mvt_field_config_editor';
+import { MVT_FIELD_TYPE } from '../../../../common/constants';
+
+test('should render field editor', async () => {
```

Comment:
The snapshot file for this test is 760 lines. I don't think you need individual test cases for each little thing. You can test errors are rendered for empty and duplicate fields in the same test. How about a single test case that tests everything to make the snapshot less redundant?

```
const fields = [
    {
      name: '',
      type: MVT_FIELD_TYPE.STRING,
    },
    {
      name: 'duplicatedFieldName',
      type: MVT_FIELD_TYPE.STRING,
    },
    {
      name: 'duplicatedFieldName',
      type: MVT_FIELD_TYPE.STRING,
    },
   {
      name: 'myValidField',
      type: MVT_FIELD_TYPE.NUMBER,
    },
  ];
```

Reply:
discussed. multiple slim test-cases testing individual BL reads better than single large test-case. but will slim-stubs to reduce snapshot footprint.